frontend/www/js/omegaup/components/problem/CollectionDetails.vue

```diff
@@ -1,20 +1,50 @@
 <template>
   <div>
     <h1 class="card-title">{{ title }}</h1>
+    <omegaup-collection-filter
+      :collection="data.collection"
+      :public-tags="data.anotherTags"
+      :selected-public-tags="selectedPublicTags"
+      @emit-add-tag="addTag"
+    ></omegaup-collection-filter>
+    <input name="selected_tags" :value="selectedTagsList" type="hidden" />
   </div>
 </template>
 
 <script lang="ts">
 import { Vue, Component, Prop } from 'vue-property-decorator';
+import collection_Filter from './Filter.vue';
 import T from '../../lang';
 import { types } from '../../api_types';
 
-@Component
+@Component({
+  components: {
+    'omegaup-collection-filter': collection_Filter,
+  },
+})
 export default class CollectionDetails extends Vue {
   @Prop() data!: types.CollectionDetailsPayload;
 
   T = T;
   type = this.data.type;
+  selectedTags = [];
```

Comment:
Aquí hay que indicar el tipo de selectedTags, de otra forma, te aparecerán errores al intentar agregar más elementos, como ocurre en L37
```suggestion
  selectedTags: types.SelectedTag[] = [];
```